packages/atomic/src/components/insight/atomic-insight-refine-modal/atomic-insight-refine-modal.tsx

```diff
@@ -54,6 +63,14 @@ export class AtomicInsightRefineModal
   private interfaceDimensions?: InsightInterfaceDimensions;
   private facetManager!: InsightFacetManager;
   private resizeObserver?: ResizeObserver;
+  private scrollCallback?: () => void;
+
+  @Listen('wheel')
+  @Listen('touchmove')
+  @Listen('keydown')
+  handleScroll() {
+    this.updateDimensions();
```

Comment:
Should there be some kind of debounce, either here or in the `updateDimensions` function ?

Does it need to run on every scroll event ? Just adding 50 or 100ms debounce might go a long way for perf.

Reply:
good point, I'll add it!